CONFIG.md

```diff
@@ -275,6 +275,55 @@ directly to
 [grunt-contrib-compass](https://github.com/gruntjs/grunt-contrib-compass)
 for this theme.
 
+### Proxying Grunt Tasks
+
+Many themes come with their own Grunt integration. In those cases you can
+configure Grunt-Drupal-Tasks for awareness of that and remove the theme
+from most handling.
+
+In order to exclude your theme from handling, specify the `proxy` property in
```

Comment:
I like brevity, but I think having consistency is important between the key name in proxy and the GDT task for which the proxy command is used.